packages/mobile/src/components/CurrencyDisplay.tsx

```diff
@@ -210,7 +210,7 @@ export default function CurrencyDisplay({
   }
 
   return (
-    <Text numberOfLines={1} style={[style, { color }]} testID={testID}>
```

Comment:
Yeah, I know it looks weird. I tried a bunch of different things to make this work without it, but if I don't add the suffix I get an object that looks like this when querying by testID:
```
      amount: {
        value: BigNumber { s: 1, e: -2, c: [Array] },
        currencyCode: 'cUSD'
      },
      formatType: 2,
      currencyInfo: { localCurrencyCode: 'MXN', localExchangeRate: '1.33' },
      testID: 'feeDrawer/SendConfirmation/totalFee',
      type: 0,
      size: 48,
      useColors: false,
      hideSign: false,
      hideSymbol: false,
      hideCode: true,
      showExplicitPositiveSign: false,
      hideFullCurrencyName: true
    }
```
which looks like the props of the `CurrencyDisplay` itself but with no `children` prop, so I can't find a way to fetch the actual text value.

This test was previously using a regex to get the item by text, but the regex was hard to read and hindered the legibility of the test imo.

It's very likely that I'm missing something simple, but I spent a few hours on this and was going a bit crazy with it, so I ended up accepting this solution. If you know why this is happening and how to fix it I'd be more than happy to remove the suffix.

Reply:
Oh weird indeed, looks like it might be this bug in `react-native-testing-library` 
https://github.com/callstack/react-native-testing-library/pull/324
We can follow up with another PR.